dashboard/app/models/script.rb

```diff
@@ -70,7 +70,7 @@ class Script < ActiveRecord::Base
   before_validation :hide_pilot_scripts
 
   def hide_pilot_scripts
-    self.hidden = true if pilot_experiment
+    self.hidden = true unless pilot_experiment.blank?
```

Comment:
Will pilot_experiment ever be something other than blank or a string for a pilot experiment?  Like could old scripts not have pilot_experiment?

Reply:
it could be `nil`, but `blank?` in Ruby on Rails means `nil` or `''`, so this would still catch the case where pilot_experiment is nil.

Reply:
Ah ok!